.gitignore

```diff
@@ -3,3 +3,4 @@ es*/compilers/*
 .DS_Store
 .ruby-version
 _site/
+.idea/
```

Comment:
blocks IntelliJ files - imo it should be in the user's global gitignore rather than an individual project's, but it can't hurt.